client/components/app.jsx

```diff
@@ -22,10 +24,23 @@ class App extends React.Component {
       });
   }
 
+  getAverage() {
+    fetch('/api/grades')
+      .then(res => res.json())
+      .then(grades => {
+        let total = 0;
+        for (let i = 0; i < grades.length; i++) {
+          total += grades[i].grade;
+        }
+        const avg = Math.trunc((total / grades.length));
+        this.setState({ average: avg });
+      });
+  }
```

Comment:
your get average should only look at `this.state.grades` and `return` an average grade instead of setting it on `state`.